core/arch/opnd.h

```diff
@@ -300,6 +300,7 @@ enum {
     DR_REG_CR13,
     DR_REG_CR14,
     DR_REG_CR15,
+    DR_REG_MAX_AS_OPSZ = DR_REG_CR15,
```

Comment:
A comment may be necessary esp since you did not immediately parse the name so it may be unclear to other readers.

Reply:
Done.